cfn-resources/cluster/cmd/resource/mappings.go

```diff
@@ -93,7 +94,7 @@ func expandReplicationSpecs(replicationSpecs []AdvancedReplicationSpec) []admin.
 	}
 
 	fmt.Printf("specs: len %d %+v", len(replicationSpecs), rSpecs)
-	return rSpecs
+	return &rSpecs
```

Comment:
similar to feedback in previous PRs, I think it's better to return rSpecs to make it clear that it can be done and we'll not be derenrenci a nil pointer, and have:
rSpecs := &[]admin.ReplicationSpec{}

Reply:
Thanks for the patience 😅
I changed it now. Seems more reasonable to always return a list and create pointer later. 

If I define it with `rSpecs := &[]admin.ReplicationSpec{}`
The append changes to `*rSpecs = append(*rSpecs, rSpec)`
This append seems a bit ugly to me, but that could just be me not being used to pointers.
WDYT?

Reply:
it's ok, your point is fair